proto/clerk/v1beta/msg.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
```

Comment:
Please lint proto files:


To format proto files: 

```bash
make proto-format
```

To check format:

```bash
make proto-lint
```

Reply:
done